src/server/game-server.js

```diff
@@ -32,10 +32,10 @@ export default class GameServer extends EventEmitter {
 
   _handleDesktopConnection(socket) {
     let d = new Date();
-    let id = hashids.encode(parseInt('1' + d.getMinutes() + '' + d.getSeconds()));
+    let id = hashids.encode(parseInt(`1${d.getMinutes()}${d.getSeconds()}`, 10));
```

Comment:
Line is too long.<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Unexpected '`'.